### DISCUSSION TITLE
Show activity from MSHV in WSJTX Window

### DISCUSSION THREAD

**Discussion by rsaue** [foldynl/QLog#170] [Ideas]
Hi!,

WSJT-X and JTDX are fully compatible when used with QLog. 

From time to time I also use MSHV http://lz2hv.org/mshv on FT8 and FT4. Logging and UDP forwarding works fine using MSHV with QLog. 

But now show of CQ traffic in QLog WSJT Window. 

I suspect this has to do with MSHV using different mode identifiers than WSJT-X and JTDX. 

This was easily solved in cqrlog a year ago, adding  a few lines to the code. 

Se my message in cqrlog forum https://www.cqrlog.com/node/3249

Could this be done in QLog too?

73 de LB2EG Richard

**Comment by foldynl** [comment]
I'll install it and try to figure out why it doesn't work. The question is, if MSHV uses the same WSJTX protocol, if this is not MSHV's issue.

**Comment by foldynl** [comment]
Richard,

Do I understand correctly that you do not see CQ spots in the QLog WSJTX CQ window when you have MSHV connected?

I have tested that CQ spots are correctly received from MSHV by QLog. I have installed HSHV v2.69 and I can see the CQ-traffic in QLog.

**Reply by rsaue** [reply]
Nothing would be better. I did my testing with av fresh install of a MSHV 2.70 beta yesterday. All was ok but no cq traffic in QLog wsjt window. I will give it a new try with in the afternoon and report back to you. 

Richard

**Reply by foldynl** [reply]
I can't find v2.70 for download. Is the beta version public?

**Reply by rsaue** [reply]
Hi,
tried again, first in windows with 0.18 and MSHV 2.69. All worked OK!

Back on Linux, i started with 0.19-dev with MSHV 2.69. The same problem as reported yesterday. nothing on display. Reverted to 0.18 without change. 

When I thereafter initiated a reply on a CQ in MSHV and aborted, the WSJTX CQ Window in QLog started to show CQ calls as it should. Changing over to 0.19 dev shows all OK there too, from startup.  Same with 0.70 beta. 

There must have been  a hickup in my  setup somewhere in Linux  not related to QLog. 

Sorry for the false alarm.

73 de LB2EG Richard
